### ISSUE TITLE
Hotkey doesn't trigger hotkey activation event for output

### ISSUE THREAD

**Issue by DaveTorrey** [nhielost/obs-midi-mg#142]
Hotkeys work if you exclusively map it as a controller input.  In other words, if you map a button to a hotkey event, it will trigger it - start, stop, toggle, etc....

If you try to map hotkeys to output to the controller (for instance to handle LEDs), however, it only works if you actually hit the keyboard hotkey, not if the controller input triggered the hotkey.

Say I have Ctrl+Shift+* to toggle something like turning a video feed on and off through a hotkey.  It will indeed toggle, but if i then try to output that event back to the controller to change the LED, it doesn't work because it was activated via the controller and not keyboard.

Long story short:  Triggering a hotkey via a controller midi mapping doesn't trigger the hotkey event for midi output events
